pkg/streamingpromql/engine_test.go

```diff
@@ -48,7 +48,6 @@ func TestUnsupportedPromQLFeatures(t *testing.T) {
 		"1 + 2":                      "scalar value as top-level expression",
 		"1 + metric{}":               "binary expression with scalars",
 		"metric{} + 1":               "binary expression with scalars",
-		"metric{} < other_metric{}":  "binary expression with '<'",
```

Comment:
We should add this back now that we're not adding support for comparison operations in this PR